Removes SN-specific docu (#316)
springernature/shunter: docs/usage/modules.md

Modules and Inheritance
=======================

> Shunter can pull in common resources, filters, helpers and templates from one or more declared npm modules.  These modules are loaded in as dependencies via `package.json` and the application is made aware of them by configuration options set in `config/local.json`.

Reviewer: 👌

> These dependencies can be overridden if there are files of the same name in your application.

Reviewer: 👌

> Code loaded in through modules can be overridden by your application by having a file of the same name in your application.

Reviewer 2: ```suggestion
These dependencies can be overridden if there are files of the same name in your application.
```
